Headlines/src/ScreenSwitcher/ScreenSwitcherViewController.swift

```diff
@@ -10,6 +10,9 @@ import UIKit
 
 class ScreenSwitcherViewController: OrientationAwareViewController {
 
+    // Property not being loaded from Storyboard.
+    // Perhaps we can use a type-safe approach to manage
+    // Storyboards and ViewControllers.
     public var initialViewController: String?
```

Comment:
write a task about this so we don't forget it

Reply:
Created #109